<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forecast</class>
 <widget class="QDialog" name="Forecast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>三维温盐流数值同化预报</string>
  </property>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>925</x>
     <y>482</y>
     <width>79</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_xw">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>216</width>
     <height>282</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yc">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>150</y>
     <width>216</width>
     <height>282</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>470</y>
     <width>160</width>
     <height>43</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>216</width>
     <height>282</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_hyf">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>216</width>
     <height>282</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
